src/main/java/org/broadinstitute/hellbender/cmdline/CommandLineProgram.java

```diff
@@ -278,6 +280,63 @@ protected void printStartupMessage(final ZonedDateTime startDateTime) {
         catch (final Exception e) { /* Unpossible! */ }
     }
 
+    /**
+     * If a tool is either Experimental or Beta, return a warning message advising against use in production environment.
+     * @return a warning message if the tool is Beta or Experimental, otherwise null
+     */
+    protected String getProductionWarning() {
+        final String KNRM = "\u001B[0m"; // reset
+        final String BOLDRED = "\u001B[1m\u001B[31m";
+
+        String warningMessage = null;
+        if (isBetaFeature()) {
+            warningMessage = String.format(
+                    "\n\n%s   %s\n\n   Warning: %s is a BETA tool and is not yet ready for use in production\n\n   %s%s\n\n",
+                    BOLDRED,
+                    Utils.dupChar('!', 60),
+                    this.getClass().getSimpleName(),
+                    Utils.dupChar('!', 60),
+                    KNRM
+            );
+        }
+        else if (isExperimentalFeature()) {
+            warningMessage = String.format(
+                    "\n\n%s   %s\n\n   Warning: %s is an EXPERIMENTAL tool and should not be used for production\n\n   %s%s\n\n",
+                    BOLDRED,
+                    Utils.dupChar('!', 60),
+                    this.getClass().getSimpleName(),
+                    Utils.dupChar('!', 60),
+                    KNRM
+            );
+        }
+        return warningMessage;
+    }
+
+    /**
+     * If a tool is either Experimental or Beta, log a warning against use in production environment.
+     */
+    protected void printProductionWarning() {
+        final String warningMessage = getProductionWarning();
+        if (warningMessage != null) {
+            logger.warn(warningMessage);
+            try {
+                Thread.sleep(5000);
```

Comment:
5 seconds of sleep is going to drive me insane.  .5 seems much more tolerable but still noticeable.  We might want to add a command line option to disable the sleep and apply that to all the tests